FrameworklessServerTests/ControllerTests.cs

```diff
@@ -84,9 +84,31 @@ public void DeleteUserShouldReturnNotFoundResponseIfUserNameIsNull()
             Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
             ResetList();
         }
-    }
 
-    public class TestUserService
-    {
+        [Fact]
+        public void PutMethodShouldReturnOkStatusIfUserSuccessfullyUpdated()
+        {
+            var controller = new Controller(_usersService);
+            var bob = new User("Bob");
+            _usersService.Add(bob);
+            Assert.Contains(_usersService.GetAllUsers(), u => u.Name == bob.Name);
+            var response = controller.UpdateUser("Bob", "Sue");
+            Assert.DoesNotContain(_usersService.GetAllUsers(), u => u.Name == bob.Name);
```

Comment:
I wouldnt worry about asserting service related code in the controller tests, just focus on what the controllers responsibility is (what response we get back)